File: ncDados/dUsuario.vb
Namespace nsUsuario

  Public Class ColecaoUsuario
    Inherits List(Of dUsuario)
  End Class

  Public Class dUsuario

    Private _cid As Nullable(Of Integer)
    Private _usuario As String
    Private _senha As String
    Private _nomeCompleto As String
    Private _situacao As String
    Private _usuarioPerfil_cid As Nullable(Of Integer)
        Private _usuarioPerfil_codigo As String
        Private _descontoProduto As Nullable(Of Decimal)
    Private _descontoPedido As Nullable(Of Decimal)
        Private _comissao As Nullable(Of Decimal)
        Private _email As String

        Public Property cid() As Nullable(Of Integer)
      Get
        Return _cid
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _cid = value
      End Set
    End Property

    Public Property usuario() As String
      Get
        Return _usuario
      End Get
      Set(ByVal value As String)
        _usuario = value
      End Set
    End Property

    Public Property senha() As String
      Get
        Return _senha
      End Get
      Set(ByVal value As String)
        _senha = value
      End Set
    End Property

    Public Property nomeCompleto() As String
      Get
        Return _nomeCompleto
      End Get
      Set(ByVal value As String)
        _nomeCompleto = value
      End Set
    End Property

    Public Property situacao() As String
      Get
        Return _situacao
      End Get
      Set(ByVal value As String)
        _situacao = value
      End Set
    End Property

    Public Property usuarioPerfil_cid() As Nullable(Of Integer)
      Get
        Return _usuarioPerfil_cid
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _usuarioPerfil_cid = value
      End Set
    End Property

        Public Property usuarioPerfil_codigo() As String
            Get
                Return _usuarioPerfil_codigo
            End Get
            Set(ByVal value As String)
                _usuarioPerfil_codigo = value
            End Set
        End Property

        Public Property descontoProduto() As Nullable(Of Decimal)
      Get
        Return _descontoProduto
      End Get
      Set(ByVal value As Nullable(Of Decimal))
        _descontoProduto = value
      End Set
    End Property

    Public Property descontoPedido() As Nullable(Of Decimal)
      Get
        Return _descontoPedido
      End Get
      Set(ByVal value As Nullable(Of Decimal))
        _descontoPedido = value
      End Set
    End Property

        Public Property comissao() As Nullable(Of Decimal)
            Get
                Return _comissao
            End Get
            Set(ByVal value As Nullable(Of Decimal))
                _comissao = value
            End Set
        End Property
        Public Property Email() As String
            Get
                Return _email
            End Get
            Set(ByVal value As String)
                _email = value
            End Set
        End Property

    End Class

End Namespace

File: ncModelos/dUsuario.cs
using System;
using System.Collections.Generic;

namespace nsModelos
{
    public class ColecaoUsuario : List<dUsuario>
    {
    }

    public class dUsuario
    {
        private int? _cid;
        private string _usuario;
        private string _senha;
        private string _nomeCompleto;
        private string _situacao;
        private int? _usuarioPerfil_cid;
        private string _usuarioPerfil_codigo;
        private decimal? _descontoProduto;
        private decimal? _descontoPedido;
        private decimal? _comissao;
        private string _email;

        public int? cid
        {
            get => _cid;
            set => _cid = value;
        }

        public string usuario
        {
            get => _usuario;
            set => _usuario = value;
        }

        public string senha
        {
            get => _senha;
            set => _senha = value;
        }

        public string nomeCompleto
        {
            get => _nomeCompleto;
            set => _nomeCompleto = value;
        }

        public string situacao
        {
            get => _situacao;
            set => _situacao = value;
        }

        public int? usuarioPerfil_cid
        {
            get => _usuarioPerfil_cid;
            set => _usuarioPerfil_cid = value;
        }

        public string usuarioPerfil_codigo
        {
            get => _usuarioPerfil_codigo;
            set => _usuarioPerfil_codigo = value;
        }

        public decimal? descontoProduto
        {
            get => _descontoProduto;
            set => _descontoProduto = value;
        }

        public decimal? descontoPedido
        {
            get => _descontoPedido;
            set => _descontoPedido = value;
        }

        public decimal? comissao
        {
            get => _comissao;
            set => _comissao = value;
        }

        public string Email
        {
            get => _email;
            set => _email = value;
        }
    }

}